run_tests.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Run tests for XBlock-utils
```

Comment:
Is it possible for you to pull out the test infrastructure into a separate PR, and set up Travis in that PR? Then you can rebase this over that and we can just add test & travis infrastructure in a logical pr.

There's some basic Travis setup here: https://github.com/edx/i18n-tools that you can probably just pull direct from.